scripts/subscribeEthereumDeposit.js

```diff
@@ -97,66 +204,200 @@ async function main (networkName, pegContractAddress) {
         }
     }
 
-    const keyring = new Keyring({type: 'sr25519'});
-    const seed = hexToU8a(process.env.CENNZNET_SECRET);
-    const claimer = keyring.addFromSeed(seed);
-    console.log('CENNZnet signer address:', claimer.address);
-
-    const spendingAssetId = await api.query.genericAsset.spendingAssetId();
+    // Keep track of latest finalized block
+    api.rpc.chain
+        .subscribeFinalizedHeads(async (head) => {
+            const blockNumber = head.number.toNumber();
+            logger.info(`HEALTH CHECK => OK`);
+            logger.info(`At blocknumber: ${blockNumber}`);
+    });
 
     const peg = new ethers.Contract(pegContractAddress, pegAbi, provider);
-    logger.info('Connecting to CENNZnet peg contract...');
-    logger.info(`CENNZnet peg deployed to: ${peg.address}`);
-    const eventConfirmation = (await api.query.ethBridge.eventConfirmations()).toNumber();
-    logger.info(`eventConfirmation::${eventConfirmation}`);
-
+    logger.info(`Connecting to CENNZnet peg contract ${pegContractAddress}...`);
+    const eventConfirmations = (await api.query.ethBridge.eventConfirmations()).toNumber();
+    let channel;
+    if (channelOverride) channel = channelOverride;
+    else channel = await rabbit.createChannel();
+    await channel.assertQueue(TOPIC_CENNZnet_CONFIRM);
+    // On eth side deposit push pub sub queue with the data, if bridge is paused, update tx status as bridge paused
     peg.on("Deposit", async (sender, tokenAddress, amount, cennznetAddress, eventInfo) => {
         logger.info(`Got the event...${JSON.stringify(eventInfo)}`);
         logger.info('*****************************************************');
-        await updateTxStatusInDB('EthereumConfirming', eventInfo.transactionHash, null, cennznetAddress);
-        const tx = await eventInfo.getTransaction();
-        await tx.wait(eventConfirmation + 1);
+        await handleDepositEvent(api, eventInfo.transactionHash, cennznetAddress, amount, tokenAddress, eventConfirmations, channel );
+    });
+}
+
+async function handleDepositEvent(api, transactionHash, cennznetAddress, amount, tokenAddress, eventConfirmations, channel ) {
+    const checkIfBridgePause = await api.query.ethBridge.bridgePaused();
+    if (!checkIfBridgePause.toHuman()) {
```

Comment:
we don't need to worry about bridge pause while depositing...
https://github.com/cennznet/bridge-contracts/pull/70
Relayer should still be able to deposit a claim..

Reply:
gotcha will update that